src/components/TaskRunsCard/index.jsx

```diff
@@ -311,9 +311,11 @@ export default class TaskRunsCard extends Component {
                 button
                 className={classes.listItemButton}
                 component={Link}
-                to={`/provisioners/${provisionerId}/worker-types/${workerType}`}
+                to={`/provisioners/${provisionerId}/worker-types/${workerType}/workers/${
+                  run.workerId
+                }`}
               >
-                <ListItemText primary="Worker Type" secondary={workerType} />
```

Comment:
The Task Details card already has the information about the Worker Type so we can remove it from the Task Runs card.